cmd/release-controller/sync_analysis.go

```diff
@@ -6,21 +6,63 @@ import (
 	"k8s.io/klog"
 )
 
+func findStatusTag(is *imagev1.ImageStream, name string) *imagev1.TagEvent {
+	for i := range is.Status.Tags {
+		tag := &is.Status.Tags[i]
+		if tag.Tag == name {
+			if len(tag.Items) == 0 {
+				return nil
+			}
+			if len(tag.Conditions) > 0 {
```

Comment:
this isn't needed, right?  If you have no conditions, then the `isTagEventblah` returns false.

Reply:
Very true.  I fixed all occurrences of this pattern.